src/ManifestStrategy/AbstractWireXmlManifestStrategy.php

```diff
@@ -0,0 +1,272 @@
+    protected const MANIFEST_KEY_NODE_NAME = 'name';
+
+    /**
+     * @var string
+     */
+    protected const MANIFEST_KEY_NODE_ATTRIBUTES = 'attributes';
+
+    /**
+     * @var string
+     */
+    protected const MANIFEST_KEY_SUB_NODES = 'sub-nodes';
+
+    /**
+     * @var \SprykerSdk\Integrator\IntegratorConfig
+     */
+    private IntegratorConfig $config;
+
+    /**
+     * @var \Symfony\Component\Filesystem\Filesystem
+     */
+    private Filesystem $filesystem;
+
+    /**
+     * @param \SprykerSdk\Integrator\IntegratorConfig $config
+     * @param \Symfony\Component\Filesystem\Filesystem $filesystem
+     */
+    public function __construct(IntegratorConfig $config, Filesystem $filesystem)
+    {
+        $this->config = $config;
+        $this->filesystem = $filesystem;
+    }
+
+    /**
+     * @param array<mixed> $manifest
+     * @param string $moduleName
+     * @param \SprykerSdk\Integrator\Dependency\Console\InputOutputInterface $inputOutput
+     * @param bool $isDry
+     *
+     * @return bool
+     */
+    public function apply(array $manifest, string $moduleName, InputOutputInterface $inputOutput, bool $isDry): bool
+    {
+        foreach ($this->config->getProjectNamespaces() as $namespace) {
+            $targetFilePath = $this->getProjectTargetXmlFilePath($namespace, $manifest[static::MANIFEST_KEY_TARGET]);
+            $rootNode = $this->getRootNode($targetFilePath, $manifest);
+            $this->addMissedElementsToProjectXmlNode($rootNode, $manifest[static::MANIFEST_KEY_DATA], $rootNode);
+
+            if (!$isDry) {
+                $this->saveNode($rootNode, $targetFilePath);
+            }
+
+            $inputOutput->writeln(sprintf('Xml manifest is applied for "%s"', $targetFilePath));
+        }
+
+        return true;
+    }
+
+    /**
+     * @param string $targetFilePath
+     * @param array $manifest
+     *
+     * @return \SimpleXMLElement
+     */
+    public function getRootNode(string $targetFilePath, array $manifest): SimpleXMLElement
+    {
+        $xmlData = $this->readFile($targetFilePath);
+
+        if ($xmlData !== null) {
+            return $this->loadXmlString($xmlData);
+        }
+
+        $rootNode = $this->loadXmlString($this->getNewFileTemplate());
+
+        $manifestRootNode = $manifest[static::MANIFEST_KEY_DATA];
+
+        if ($rootNode->getName() !== $manifestRootNode[static::MANIFEST_KEY_NODE_NAME]) {
+            return $rootNode;
+        }
+
+        foreach ($manifestRootNode[static::MANIFEST_KEY_NODE_ATTRIBUTES] ?? [] as $attribute => $value) {
+            $rootNode->addAttribute($attribute, $value);
+        }
+
+        return $rootNode;
+    }
+
+    /**
+     * @param \SimpleXMLElement|null $projectNode
+     * @param array $manifestNode
+     * @param \SimpleXMLElement $parentNode
+     *
+     * @return void
+     */
+    public function addMissedElementsToProjectXmlNode(?SimpleXMLElement $projectNode, array $manifestNode, SimpleXMLElement $parentNode): void
```

Comment:
the `ext-simplexml` extension must be added to the composer.json

Reply:
done